qemu/tests/numa_cpu.py

```diff
@@ -43,21 +45,39 @@ def _get_boundary(value, max_value, weight):
 
     def numa_cpu_guest():
         """
-        Get the cpu id list for each node in guest os, linux only
+        Get the cpu id list for each node in guest os, sort with node id.
         """
         error_context.context("Get cpus in guest os", logging.info)
-        numa_info_guest = NumaInfo(session=session)
-        nodes_guest = numa_info_guest.online_nodes
         numa_cpu_guest = []
-        for node in nodes_guest:
-            numa_cpus = numa_info_guest.online_nodes_cpus[node]
-            numa_cpus = set([int(v) for v in numa_cpus.split()])
-            numa_cpu_guest.append(numa_cpus)
+        if vm_arch in ('ppc64', 'ppc64le'):
+            numa_info_guest = NumaInfo(session=session)
+            nodes_guest = numa_info_guest.online_nodes
+            for node in nodes_guest:
+                numa_cpus = numa_info_guest.online_nodes_cpus[node]
+                numa_cpus = sorted([int(v) for v in numa_cpus.split()])
+                numa_cpu_guest.append(numa_cpus)
+        else:
+            error_context.context("Get SART ACPI table", logging.info)
+            if not utils_package.package_install("acpidump", session):
+                test.cancel("Please install acpidump in guest to proceed")
+            content = session.cmd('cd /tmp && acpidump -b && iasl -d srat.dat'
```

Comment:
session.cmd_output?

Reply:
> session.cmd_output?

Both can work, `cmd` return the output and raise exception if the exit status is nonzero. 
I just do not want to break `iasl -d srat.dat` :)

Reply:
Updated ~   thanks!